common/persistence/persistence-tests/persistenceTestBase.go

```diff
@@ -439,7 +440,8 @@ func (s *TestBase) CreateChildWorkflowExecution(ctx context.Context, domainID st
 			TimerTasks:       timerTasks,
 			VersionHistories: versionHistories,
 		},
-		RangeID: s.ShardInfo.RangeID,
+		RangeID:    s.ShardInfo.RangeID,
+		DomainName: s.DomainManager.GetName(),
```

Comment:
nit: The DomainName set here is not the correct value. But since this is the test code, and we don't care about the metrics in test, it's ok to not add domainName in test code.